docs/spec/staking/spec-technical.md

```diff
@@ -154,76 +169,82 @@ Description:
  - AdjustmentFeePool: Adjustment factor used to passively calculate each bonds
    entitled fees from `GlobalState.FeePool`
  - AdjustmentRewardPool: Adjustment factor used to passively calculate each
-   bonds entitled fees from `Candidate.ProposerRewardPool``
+   bonds entitled fees from `Candidate.ProposerRewardPool`
+      
  
 ### QueueElem
 
 Unbonding and re-delegation process is implemented using the ordered queue data structure. 
-All queue elements used share a common structure:
+All queue elements share a common structure:
```

Comment:
I think that we decided to reorganize this for the spec.. to not include `QueueElem` it will be an implementation detail whether we choose to have a common struct or now... this comment means we should move all the fields of QueueElem into any other struct in the spec which references the QueueElem